synthesizer/exceptions.py

```diff
@@ -165,3 +165,20 @@ def __str__(self):
             return '{0} '.format(self.message)
         else:
             return 'Image not yet created!'
+
+        
+class WavelengthOutOfRange(Exception):
+    """
+    Exception class for when an image has not yet been made
```

Comment:
this docstring doesn't correspond to the error

Reply:
Ooops...